### PR TITLE
library: Add WebView demo in Rust

### PR DESCRIPTION
**Pull request by Hofer-Julian**
- Ensure types of WebKitGtk 6.0 in previewer.vala
- Add WebView demo

### CODE REVIEW

**Review by M-Sabrina — APPROVED** [review]

**Comment by Hofer-Julian** [comment]
Since CI also fails at https://github.com/sonnyp/Workbench/pull/671, I am going ahead and merge this.

File: src/Previewer/previewer.vala [hunk 1]
```diff
@@ -59,10 +59,15 @@ namespace Workbench {
       return true;
     }
 
-    public void update_ui (string content, string target_id, string original_id = "") {
+    // This registers GObjects so that they can be found by GtkBuilder
+    void ensure_types() {
```
Comment (lw64) [inline src/Previewer/previewer.vala:63]: This should have the `private` modifier.

Comment (Hofer-Julian) [reply src/Previewer/previewer.vala:63]: Private is not the default?
Then why do we put public on all the other functions?

Comment (lw64) [reply src/Previewer/previewer.vala:63]: It is, but it should be explicit.

**Pull request merged**